tests/types/test_get_operation_type.py

```diff
@@ -0,0 +1,111 @@
+import pytest

```

Comment:
Hi @Speedy1991, is there a particular reason why the test file is located in `tests/types` instead of in `tests/utils` next to `test_get_first_operation.py` ?

Reply:
Haha, to be honest, I didn’t find a matching test file - I completely overlooked test_get_first_operation.py
I'll move the test